Chirp/Pages/RegistrationPage/RegistrationViewModel.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  RegistrationViewModel.swift
+//  Chirp
+//
+//  Created by Stepan Kukharskyi on 6/9/23.
+//
+
+import UIKit
+import Combine
+import FirebaseAuth
+
+class RegistrationViewModel {
+    
+    private var phoneNumber: String
+    private var imageExtension: String = "jpg"
```

Comment:
General rule of thumb... if the data doesn't change, make it a constant (Constant by default)